.codegen/service.go.tmpl

```diff
@@ -19,20 +19,42 @@ import (
 {{end}}
 
 {{define "service"}}
-var Cmd = &cobra.Command{
-	Use:   "{{(.TrimPrefix "account").KebabName}}",
-	{{- if .Description }}
-	Short: `{{.Summary | without "`"}}`,
-	Long: `{{.Comment  "  " 80 | without "`"}}`,
-	{{- end }}
-	Annotations: map[string]string{
-		"package": "{{ .Package.Name }}",
-	},
-	{{- if .IsPrivatePreview }}
+// Slice with functions to override default command behavior.
+// Functions can be added from the `init()` function in manually curated files in this directory.
+var cmdOverrides []func(*cobra.Command)
 
-	// This service is being previewed; hide from help output.
-	Hidden: true,
-	{{- end }}
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "{{(.TrimPrefix "account").KebabName}}",
+		{{- if .Description }}
+		Short: `{{.Summary | without "`"}}`,
+		Long: `{{.Comment  "  " 80 | without "`"}}`,
+		{{- end }}
+		GroupID: "{{ .Package.Name }}",
```

Comment:
This is moved here from the init() methods for `cmd/workspace/cmd.go` and `cmd/account/cmd.go`?